src/runner/centralized_kb_hook.cpp

```diff
@@ -143,14 +351,35 @@ namespace CentralizedKeyboardHook
             return CallNextHookEx(hHook, nCode, wParam, lParam);
         }
 
+        LocalKey lHotkey;
+
+        lHotkey.win = (GetAsyncKeyState(VK_LWIN) & 0x8000) || (GetAsyncKeyState(VK_RWIN) & 0x8000);
```

Comment:
I think by introducing a new `LocalKey` class and querying key state here we might be circumventing and duplicating a lot of KBM functionality, i.e. those modifiers could be remapped in some way etc. 

Is there any reason why we want to add the logic to run programs to the centralized hook here as opposed to moving to `KeyboardEventHandlers.cpp`?